static/app/components/pageTimeRangeSelector.tsx

```diff
@@ -1,64 +1,32 @@
 import styled from '@emotion/styled';
 
-import TimeRangeSelector from 'sentry/components/organizations/timeRangeSelector';
-import Panel from 'sentry/components/panels/panel';
+import {TimeRangeSelector} from 'sentry/components/timeRangeSelector';
 import {t} from 'sentry/locale';
-import {space} from 'sentry/styles/space';
 
-type Props = React.ComponentProps<typeof TimeRangeSelector> & {className?: string};
+type PageTimeRangeSelectorProps = React.ComponentProps<typeof TimeRangeSelector>;
 
-function PageTimeRangeSelector({className, ...props}: Props) {
+function PageTimeRangeSelector(props: PageTimeRangeSelectorProps) {
   return (
-    <DropdownDate className={className}>
-      <TimeRangeSelector
-        key={`period:${props.relative}-start:${props.start}-end:${props.end}-utc:${props.utc}-defaultPeriod:${props.defaultPeriod}`}
-        label={<DropdownLabel>{t('Date Range:')}</DropdownLabel>}
-        detached
-        {...props}
-      />
-    </DropdownDate>
+    <StyledTimeRangeSelector
+      key={`period:${props.relative}-start:${props.start}-end:${props.end}-utc:${props.utc}-defaultPeriod:${props.defaultPeriod}`}
+      triggerProps={{icon: null, prefix: t('Date Range')}}
+      // Use relative option labels (e.g. "Last 7 days") not keys ("7D")
+      triggerLabel={
+        props.relativeOptions && props.relative
+          ? props.relativeOptions[props.relative]
+          : null
+      }
+      menuWiderThanTrigger
+      {...props}
+    />
   );
 }
 
-const DropdownDate = styled(Panel)`
-  padding: 0;
-  margin: 0;
+const StyledTimeRangeSelector = styled(TimeRangeSelector)`
+  width: 100%;
 
-  display: flex;
-  justify-content: center;
-  align-items: center;
-  height: ${p => p.theme.form.md.height}px;
-  background: ${p => p.theme.background};
-  border: 1px solid ${p => p.theme.border};
-  border-radius: ${p => p.theme.borderRadius};
-
-  font-size: ${p => p.theme.fontSizeMedium};
-  color: ${p => p.theme.textColor};
-
-  > div {
+  button[aria-haspopup='listbox'] {
     width: 100%;
```

Comment:
this isn't ideal — it's error-prone plus I think the trigger button shouldn't span full width — but let's just keep it here to not disturb the UI more than necessary